.github/release-please/release-please-config.json

```diff
@@ -6,18 +6,8 @@
       "include-component-in-tag": false,
       "changelog-sections": [
         { "type": "feat", "section": "Features" },
-        { "type": "fix", "section": "Bug Fixes" },
-        { "type": "perf", "section": "Performance Improvements" },
-        { "type": "revert", "section": "Reverts" },
-        { "type": "internal", "section": "Internal Changes" },
-        { "type": "deps", "section": "Dependencies" },
-        { "type": "docs", "section": "Documentation" },
-        { "type": "chore", "section": "Miscellaneous Chores" },
-        { "type": "refactor", "section": "Code Refactoring" },
```

Comment:
We also use `chore` and `refactor`, so we might wanna add them as well.

Reply:
Ah I missed this, I thought these were not being included in the changelog/release notes. Since you are already using these in your workflow and are included in the release I think it makes sense to add them in this case. See [d2c69a1](https://github.com/grafana/synthetic-monitoring-app/pull/933/commits/d2c69a16b37fc95eb69571e881e087744f54a2a4).